config/initializers/ruby_money.rb

```diff
@@ -1,8 +1,28 @@
 # frozen_string_literal: true
 
+require 'money/bank/currencylayer_bank'
+
 Money.locale_backend = :i18n
 
-eu_bank = EuCentralBank.new
-Money.default_bank = eu_bank
+if Rails.env.test? || Rails.env.development?
+  eu_bank = EuCentralBank.new
+  Money.default_bank = eu_bank
+else
+  mclb = Money::Bank::CurrencylayerBank.new
+  mclb.access_key = AppSecrets.CURRENCY_LAYER_API_KEY
+  mclb.currencylayer = true
+  mclb.ttl_in_seconds = 86_400
+  mclb.cache = proc.new do |payload|
```

Comment:
Shouldn't this be `Proc.new` (note the capitalization)?

Reply:
Actually rubocop asked to replace `Proc.new` with `proc`. After re-reading again, I realized it's `proc` and not `proc.new`. I cannot test it right now as my access key has expired. Once the budget is approved, I'll test again using a new test key created with WCA's account.